interfaces/dotnet/Cantera.sln

```diff
@@ -0,0 +1,34 @@
+﻿
+Microsoft Visual Studio Solution File, Format Version 12.00
+# Visual Studio Version 16
+VisualStudioVersion = 16.0.30114.105
```

Comment:
Can you explain / comment what this file does? I gather that it isn't just for use with Visual Studio.

Reply:
The solution file format is used to group related C# projects. It is supported by the cross-platform, open-source dotnet CLI; the open-source OmniSharp framework which underpins editor plugins for VS Code, Vim, etc; and, yes, Visual Studio. It is not meant to be manually edited. Admittedly, it has a rather bizarre, idiosyncratic format.

There is no assumption that any contributor would use Visual Studio; rather, they should use whatever editor they prefer. The solution file is a convenience to help their editor plugin if they use one.

Reply:
The question I have is how tied is the utility of this file to the very specific version of VS in the file?

Reply:
Honestly I have no idea why a version number appears in there! I just typed `dotnet new sln` and it created the solution file. It appears though Format Version 12 is used since VS 2013 and by extension any of the .NET SDKs that support the solution file.